packages/material-ui/src/Switch/Switch.js

```diff
@@ -215,20 +215,21 @@ const Switch = React.forwardRef(function Switch(inProps, ref) {
   };
 
   const classes = useUtilityClasses(styleProps);
-
   const icon = <SwitchThumb className={classes.thumb} styleProps={styleProps} />;
 
   return (
-    <SwitchRoot className={clsx(classes.root, className)} styleProps={styleProps}>
+    <SwitchRoot className={clsx(classes.root, className)} sx={sx} styleProps={styleProps}>
       <SwitchSwitchBase
-        className={classes.switchBase}
```

Comment:
Is this not supported? Seems a lot simpler compared to props spread.

Reply:
className is supported, I used classes.root to save a few bytes, but no strong preference

Reply:
I understand. So now consistency doesn't matter, gotcha.

> I have used classes.root instead to save a few bytes

It doesn't save bytes.

Reply:
On the consistency side of things, it depends on what we consider. For instance, here, we only leverage the classes https://github.com/mui-org/material-ui/blob/76d72cc19cf3db761a8947d74831ad67da3ef97d/packages/material-ui/src/OutlinedInput/OutlinedInput.js#L155-L158 but no real preference, we co do a follow-up to harmonize the codebase.

For the reducing of bundle size, I mean that the alternative is:

```diff
diff --git a/packages/material-ui/src/Switch/Switch.js b/packages/material-ui/src/Switch/Switch.js
index 11d03ef22e..7b08055aa1 100644
--- a/packages/material-ui/src/Switch/Switch.js
+++ b/packages/material-ui/src/Switch/Switch.js
@@ -222,13 +222,14 @@ const Switch = React.forwardRef(function Switch(inProps, ref) {
       <SwitchSwitchBase
         type="checkbox"
         icon={icon}
+        className={classes. switchBase}
         checkedIcon={icon}
         ref={ref}
         styleProps={styleProps}
         {...other}
         classes={{
           ...classes,
-          root: classes.switchBase,
+          root: null,
         }}
       />
       <SwitchTrack className={classes.track} styleProps={styleProps} />
```

Not a significant difference, we can probably ignore it.